builtin/builtin.mbti

```diff
@@ -257,6 +257,8 @@ fn[K : Hash + Eq, V] Map::contains(Self[K, V], K) -> Bool
 fn[K : Hash + Eq, V : Eq] Map::contains_kv(Self[K, V], K, V) -> Bool
 fn[K, V] Map::each(Self[K, V], (K, V) -> Unit?Error) -> Unit?Error
 fn[K, V] Map::eachi(Self[K, V], (Int, K, V) -> Unit?Error) -> Unit?Error
```

Comment:
in general we want to one PR focusing on one issue
it seems this PR adding new functions is mixed with adding error polymorphism cc @peter-jerry-ye